test/typecheck/func_def_param.exp

```diff
@@ -0,0 +1,27 @@
+ProgramNode
+  FuncDefNode add: int
+    ParamNode
+      a: int
+    ParamNode
+      b: int
+    ParamNode
+      c: int
```

Comment:
```suggestion
    ParamNode a: int
    ParamNode b: int
    ParamNode c: int
```

Since the `id` isn't another AST node, it's better to place them after their node instead of creating a separate indentation block.